common/sgx/eeid.c

```diff
@@ -356,19 +356,21 @@ oe_result_t oe_remeasure_memory_pages(
 
     if (with_eeid_pages)
     {
+        char* eeid_bytes = (char*)eeid;
         size_t num_bytes = oe_eeid_byte_size(eeid);
         size_t num_pages =
             num_bytes / OE_PAGE_SIZE + ((num_bytes % OE_PAGE_SIZE) ? 1 : 0);
 
-        oe_page_t* pages = malloc(sizeof(oe_page_t) * num_pages);
-        memset(pages, 0, sizeof(oe_page_t) * num_pages);
-        memcpy(pages->data, eeid, num_bytes);
-
+        oe_page_t* page = malloc(sizeof(oe_page_t));
```

Comment:
This reminds me, for this change to be compatible with upcoming Linux kernel changes, all allocations we `_add_page` into enclave need to be page aligned, see #2566.
```suggestion
        oe_page_t* page = oe_memalign(OE_PAGE_SIZE, sizeof(oe_page_t));
        if (!page)
            OE_RAISE(OE_OUT_OF_MEMORY);
```

Reply:
This is in the EEID re-measuring of memory pages, it doesn't actually add or remove anything, and it's in `common`. I renamed `_add_page` to `_remeasure_page` to make that clearer. The memory allocation was unnecessary, I replaced it with an `oe_page_t` on the stack.

Reply:
Yes, the use of `_add_page` matching what we call in create.c was indeed the source of my confusion, thank you for clarifying and renaming!